apps/documentation/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@kiwicom/margarita-docs",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "start": "next -p 1337",
+    "build": "next build && next export"
+  },
+  "dependencies": {
+    "@mdx-js/mdx": "^0.15.0",
+    "mdx-docs": "^2.0.0-alpha.0",
+    "next": "^7.0.1",
+    "next-images": "^1.0.4",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "remark-autolink-headings": "^5.1.0",
+    "remark-emoji": "^2.0.2",
+    "remark-images": "^0.16.1",
+    "remark-slug": "^5.1.1",
+    "remark-unwrap-images": "^0.1.0"
```

Comment:
all dependencies here that are defined in other package.json should be moved to `peerDependencies`

Should be `next`, `react`, `react-dom` atleast